cmd/operator-sdk/bundle/create.go

```diff
@@ -230,11 +232,21 @@ func (c bundleCreateCmd) runGenerate() error {
 		c.overwrite = true
 	}
 
+	annotationsFile := getAnnotationsFilePath(c.outputDir, c.directory)
+
+	if isExist(annotationsFile) && !c.overwrite {
+		err := registry.RemoveSDKstampsInAnnotation(annotationsFile)
```

Comment:
You shouldn't need to remove these if you're writing labels to `bundle.Dockerfile` too.

Reply:
Fixed by https://github.com/operator-framework/operator-registry/pull/358.